src/App.js

```diff
@@ -31,30 +31,29 @@ const App = () => {
       <ErrorBoundary>
         <UserProvider>
           <HelmetProvider>
```

Comment:
I'm basically wrapping everything in this provider. Is it how it's supposed to be used? 🙈